client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/CRUDDocumentationIT.java

```diff
@@ -746,7 +746,7 @@ public void onFailure(Exception e) {
             listener = new LatchedActionListener<>(listener, latch);
 
             // tag::bulk-execute-async
-            client.bulkAsync(request, listener); // <1>
+            client.bulkAsync(request, RequestOptions.DEFAULT, listener); // <1>
```

Comment:
Should the docs changes go back to 6.x?

Reply:
yes I had already made them with the other PR but some were left behind, I will cherry-pick them.

Reply:
Thanks!